RecastDemo/Source/TestCase.cpp

```diff
@@ -102,9 +102,22 @@ bool TestCase::load(const std::string& filePath)
 	FILE* fp = fopen(filePath.c_str(), "rb");
 	if (!fp)
 		return false;
-	fseek(fp, 0, SEEK_END);
+	if (fseek(fp, 0, SEEK_END) != 0)
+	{
+		fclose(fp);
+		return false;
+	}
 	int bufSize = ftell(fp);
```

Comment:
I think we should change this to be of type `long` now that you're making changes around here. On 64-bit Linux distros I seem to recall that `long` is frequently 8 bytes and `int` is 4, and `ftell` returns `long`.